rllib/algorithms/algorithm.py

```diff
@@ -2810,8 +2810,14 @@ def default_resource_request(
 
         # resources for remote learner workers
         learner_bundles = []
-        if cf.enable_rl_module_and_learner and cf.num_learners > 0:
-            learner_bundles = cls._get_learner_bundles(cf)
+        if cf.enable_rl_module_and_learner:
+            if cf.num_learners > 0:
+                learner_bundles = cls._get_learner_bundles(cf)
+            # Aggregation actors (for the local learner).
+            else:
+                learner_bundles = [
+                    {"CPU": 1} for _ in range(cf.num_aggregator_actors_per_learner)
```

Comment:
Ah yeah, makes sense. I was thinking that the GPU LoaderThreads are now requesting CPUs. I had 2 agregators and 2 x {CPU: 1} placement groups were missing. Awesome fix!